egs/tedlium/s5/local/score_sclite.sh

```diff
@@ -48,6 +49,15 @@ nj=$(cat $dir/num_jobs)
 
 mkdir -p $dir/scoring/log
 
+if [ -f $dir/../frame_shift ]; then
```

Comment:
I am not familiar with this part. @vimalmanohar could you please review this script.